### DISCUSSION TITLE
紫竹的相册

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#118] [Announcements]
# 紫竹的相册

https://youngjuning.js.org/album/

**Comment by pandalabme** [comment]
相册很赞呀！
请问大佬的主题是基于halo-theme-butterfly二次开发的吗？好像目前[官方](https://github.com/dhjddcn/halo-theme-butterfly/issues/79)还不支持图库功能..

**Reply by youngjunning** [reply]
这个，官方支持的，我也是按照官网示例搞得

**Reply by pandalabme** [reply]
感谢大佬的回复！
我本来想使用[Halo 2.0 的相册管理插件](https://github.com/halo-sigs/plugin-photos)来实现，但这个插件需要主题提供模板（photos.html）才行，在[halo-theme-butterfly](https://github.com/dhjddcn/halo-theme-butterfly)上没找到..
我再去官网示例研究研究

**Reply by youngjunning** [reply]
这个看起来是移植的，比对下版本是不是不对。我是 hexo 用户
